hypha/apply/funds/models/applications.py

```diff
@@ -364,6 +366,9 @@ def serve(self, request, *args, **kwargs):
 
                 if form.is_valid():
                     form_submission = self.process_form_submission(form, draft=draft)
+                    site = Site.find_for_request(request)
+                    application_settings = ApplicationSettings.for_site(site=site)
+                    extra_text = application_settings.extra_text_round
```

Comment:
@Parbhat I want this extra text variable to be available in the template, see below. How does one make that happen in the best way?

Reply:
Settings can be accessed in templates directly with settings context variable. More in [docs](https://docs.wagtail.io/en/stable/reference/contrib/settings.html#using-in-django-templates) and [example](https://github.com/OpenTechFund/hypha/blob/174a0317e20416fd94f0d731f673e2bb64999c34/hypha/apply/utils/templates/apply/404.html#L10).

Reply:
Thanks! This I should have remembered.